README.md

```diff
@@ -586,6 +586,18 @@ Export node SSH key. Valid values are 'present' and 'absent'.
 
 - *Default*: 'present'
 
+ssh_key_export
+--------------
+Controls whether node SSH host key should be exported. Valid values are 'true and 'false'.
```

Comment:
Could you please change to `Boolean to determine if the node's SSH host key should be exported.`

I think that's a bit clearer and encourages booleans as opposed to stringified booleans.